packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag

```diff
@@ -711,7 +777,9 @@ void main() {
 
     vec3 irradiance = getAmbientLightIrradiance( ambientLightColor );
 
-    #if THREE_VRM_THREE_REVISION >= 133
+    #if THREE_VRM_THREE_REVISION >= 157
+      irradiance += getLightProbeIrradiance( lightProbe, geometryNormal );
```

Comment:
(premise: https://github.com/pixiv/three-vrm/pull/1304/files#r1342518857)

so, we want to use `#if defined( USE_LIGHT_PROBES )` here.
Having the macro `#if THREE_VRM_THREE_REVISION >= 157` looks good

Reply:
commit: https://github.com/pixiv/three-vrm/pull/1304/commits/8088de152e58a486622b9aff8664a7dc342d2af3